content/id/docs/concepts/extend-kubernetes/compute-storage-net/network-plugins.md

```diff
@@ -0,0 +1,159 @@
+---
+title: Plugin Jaringan
+content_template: templates/concept
+weight: 10
+---
+
+
+{{% capture overview %}}
+
+{{< feature-state state="alpha" >}}
+{{< warning >}}Fitur-fitur Alpha berubah dengan cepat. {{< /warning >}}
+
+Network plugins in Kubernetes come in a few flavors:
+_Plugin_ jaringan di Kubernet datang dalam beberapa varian:
+
+* _Plugin_ CNI : mengikuti spesifikasi appc / CNI, yang dirancang untuk interoperabilitas.
+* _Plugin_ Kubenet : mengimplementasikan dasar `cbr0` menggunakan _plugin_ `bridge` dan `host-local` CNI
```

Comment:
```suggestion
* _Plugin_ Kubenet : mengimplementasi `cbr0` sederhana  menggunakan _plugin_ `bridge` dan `host-local` CNI
```

- Basic `cbr0` -> I think it means `cbr0` sederhana, instead on "dasar `cbr0`" which actually means "the base of `cbr0`"
- Implements -> Mengimplementasi instead of "mengimplementasikan"

Reply:
done